docs/wip/exchange-onboarding.md

```diff
@@ -0,0 +1,352 @@
+    enclosed in double quotes.
+-   **`topologyFile`:** Used to connect to a custom set of nodes on
+    the network. When unspecified an appropriate
+    default topology is generated.
+
+## How do I export the CA certificate for the API?
+
+The certificate is generated inside the wallet in the file `tls/server.cert`
+
+If you are using the docker container, this can be output using the command:
+
+`docker exec -it cardano-mainnet-wallet cat /wallet/state-wallet-mainnet/tls/server.cert`
+
+Please refer to your OS or browser documentation for how to import the CA
+certificate into your trusted `ca-certificates` file. The rest of this
+document will assume the certificate is trusted.
+
+## How do I know when the wallet has fetched all the blocks?
+
+You can check the sync progress using the API to get detailed json output:
+
+    curl -X GET "https://127.0.0.1:8090/api/v1/node-info" -H "accept: application/json;charset=utf-8"
+    {"data":{"syncProgress":{"quantity":100,"unit":"percent"},"blockchainHeight":{"quantity":738268,"unit":"blocks"},"localBlockchainHeight":{"quantity":738268,"unit":"blocks"},"localTimeDifference":{"quantity":0,"unit":"microseconds"}},"status":"success","meta":{"pagination":{"totalPages":1,"page":1,"perPage":1,"totalEntries":1}}}
+
+The following command can be used to see the percentage completion of the sync only:
+
+    nix-shell -p jq curl --run 'curl -X GET "https://127.0.0.1:8090/api/v1/node-info" -H "accept: application/json;charset=utf-8" | jq .data.syncProgress.quantity'
+
+## Creating a New Wallet
+You can create your first wallet using the POST /api/v1/wallets endpoint as follow:
+
+    curl -X POST https://localhost:8090/api/v1/wallets                     \
+         -H "Content-Type: application/json; charset=utf-8"                \
+         -H "Accept: application/json; charset=utf-8"                      \
+         --cacert ./scripts/tls-files/ca.crt                               \
+         -d '{                                                             \
+      "operation": "create",                                               \
+      "backupPhrase": ["squirrel", "material", "silly", "twice", "direct", \
+        "slush", "pistol", "razor", "become", "junk", "kingdom", "flee"],  \
+      "assuranceLevel": "normal",                                          \
+      "name": "MyFirstWallet"                                              \
+    }'
+Warning: Those 12 mnemonic words given for the backup phrase act as an example. Do not use them on a production system. See the section below about mnemonic codes for more information.
+
+As a response, the API provides you with a wallet id used in subsequent requests to uniquely identity the wallet. Make sure to store it / write it down. Note that every API response is jsend-compliant; Cardano also augments responses with meta-data specific to pagination. More details in the section below about Pagination.
```

Comment:
> wallet id used in subsequent requests to uniquely identity the wallet

Slightly less wordy: "unique wallet ID to be used in subsequent requests"

Reply:
> jsend-compliant

Can you link to reference material for `jsend`? I hadn't heard of it until now.

Reply:
> More details in the section below about Pagination

I don't see it. Please add one.

Reply:
I wonder if I should just remove the mention of pagination, or do you think it's relevant to include that in the FAQ?

Reply:
Is it described well enough in the API reference docs? If so, you can just refer the reader to that.